qmi/tools/proc.py

```diff
@@ -537,6 +537,21 @@ def shutdown_context(context_name: str, progressfn: Callable[[str], None]) -> Sh
     :return: Tuple (responding, pid, success).
     :raises ProcessException: If an error occurs.
     """
+    def wait_disappear() -> bool:
+        """Wait a context to disappear.
```

Comment:
minor: Wait a context -> Wait for a context

Reply:
Fixing the grammar